packages/destination-actions/src/destinations/amplitude/index.ts

```diff
@@ -68,19 +69,36 @@ const destination: DestinationDefinition<Settings> = {
           'Amplitude project secret key. You can find this key in the "General" tab of your Amplitude project.',
         type: 'string',
         required: true
+      },
+      endpoint: {
+        label: 'Endpoint Region',
+        description: 'The region to send your data.',
+        type: 'string',
+        format: 'text',
+        choices: [
+          {
+            label: 'North America',
+            value: 'north_america'
+          },
+          {
+            label: 'Europe',
+            value: 'europe'
+          }
+        ],
+        default: 'north_america'
       }
     },
     testAuthentication: (request, { settings }) => {
       // Note: Amplitude has some apis that use basic auth (like this one)
       // and others that use custom auth in the request body
-      return request('https://amplitude.com/api/2/usersearch?user=testUser@example.com', {
+      return request(getEndpointByRegion('usersearch', settings.endpoint) + '?user=testUser@example.com', {
```

Comment:
To make this a bit more readable:

```suggestion
      const endpoint = getEndpointByRegion('usersearch', settings.endpoint) 
      return request(`${endpoint}?user=testUser@example.com`, {
```